packages/react-charting/src/utilities/colors.ts

```diff
@@ -90,18 +90,18 @@ const QualitativePalette: Palette = {
   '31': ['#2c3c85', '#c8d1fa'], // [cornflower.shade30, cornflower.tint40],
   '32': ['#7f004e', '#f7adda'], // [hotPink.shade30, hotPink.tint40],
   '33': ['#02494c', '#9bd9db'], // [teal.shade30, teal.tint40],
-  '34': ['#4c3867', '#d7caea'], // [orchid.shade30, orchid.tint40],
+  '34': ['#4c3867', '#b29ad4'], // [orchid.shade30, orchid.tint30],
   '35': ['#0b5a08', '#a7e3a5'], // [lightGreen.shade30, lightGreen.tint40],
   '36': ['#20547c', '#83bdeb'], // [lightBlue.shade30, lightBlue.tint30],
   '37': ['#712d09', '#df8e64'], // [pumpkin.shade30, pumpkin.tint30],
   '38': ['#23330b', '#a4cc6c'], // [lime.shade40, lime.tint30],
   '39': ['#63276d', '#cf87da'], // [lilac.shade30, lilac.tint30],
-  '40': ['#3a2f00', '#ecdfa5'], // [gold.shade40, gold.tint40],
+  '40': ['#3a2f00', '#dac157'], // [gold.shade40, gold.tint30],
 };
 
 const SemanticPalette: Palette = {
   info: ['#015cda'],
-  disabled: ['#8a8886'],
+  disabled: ['#dbdbdb', '#4d4d4d'], // [grey[86], grey[30]]
```

Comment:
disabled light is #bdbdbd

Reply:
Inspect and the tokens package says it should be #dbdbdb

![image](https://user-images.githubusercontent.com/110246001/223915922-7ba7166b-2ab0-4e1f-92c9-4e90ab357d8e.png)

![image](https://user-images.githubusercontent.com/110246001/223916108-76c3b0b4-8ea3-4505-90e7-a7822ee62ad9.png)